erpnext/hr/doctype/expense_claim_detail/expense_claim_detail.json

```diff
@@ -223,40 +326,80 @@
    "width": "150px"
   }, 
   {
+   "allow_bulk_edit": 0, 
    "allow_on_submit": 0, 
    "bold": 0, 
    "collapsible": 0, 
+   "columns": 2, 
+   "fieldname": "tax_amount", 
```

Comment:
I don't want to do tax percentage here, because typically when you have expenses you have just the total amount and the tax amount. If the user has to calculate percentages every time, it gets to be pretty complicated and unusable. 

The only time the % is important is if the sanctioned amount is not the full amount.